README.md

```diff
@@ -13,16 +13,16 @@ To run the example project, clone the repo, and run `pod install` from the Examp
 
 ## Installation
 
-CYFBeacon is available through [CocoaPods](http://cocoapods.org). To install
+RBIBeacon is available through [CocoaPods](http://cocoapods.org). To install
 it, simply add the following line to your Podfile:
 
-    pod "CYFBeacon"
+    pod "CYFBeacon", :git => 'https://github.com/robinpowered/CYFBeacon.git'
 
 ## Author
```

Comment:
Let's just change this to Contributors...definitely leave Victor and you can add yourself since youre maintaining now.